modules/embedding-microshift-into-rpm-ostree-image.adoc

```diff
@@ -0,0 +1,212 @@
+// Module included in the following assemblies:
```

Comment:
Start all modules with microshift-
For example, microshift-embedding-into-rpm-ostree-image.doc